migration_sql_test.go

```diff
@@ -113,14 +116,44 @@ func TestUseTransactions(t *testing.T) {
 		if err != nil {
 			t.Error(err)
 		}
-		_, useTx := getSQLStatements(f, true)
+		_, useTx, err := getSQLStatements(f, true)
+		if err != nil {
+			t.Error(err)
+		}
 		if useTx != test.useTransactions {
 			t.Errorf("Failed transaction check. got %v, want %v", useTx, test.useTransactions)
 		}
 		f.Close()
 	}
 }
 
+func TestParsingErrors(t *testing.T) {
+	type testData struct {
+		sql   string
+		error bool
+	}
+	tests := []testData{
+		{
+			sql:   statementBeginNoStatementEnd,
+			error: true,
+		},
+		{
+			sql:   unfinishedSQL,
+			error: true,
+		},
+		{
+			sql:   noUpDownAnnotations,
+			error: true,
+		},
+	}
+	for _, test := range tests {
+		_, _, err := getSQLStatements(strings.NewReader(test.sql), true)
+		if err == nil {
```

Comment:
did you mean to use test.error here?

Reply:
Yes, that is the desired result.

Open to changing it to something else if you have anything in mind?

Reply:
The code from how you're defining the tests reads like you're expecting either error or no error, in which case you should be using the test.error bool to conditionally check for whether err is nil. 

However, every case is `error: true` and the error print below may be unclear since you're saying you want `true` instead of `nil`.

Reply:
The feature works well, I've tested it with the example project. After cleaning up this test code I think this is good to go 👍 @VojtechVitek